### PR TITLE
roslyn model addition (part of DCP220387)

### PR DESCRIPTION
**Pull request by PedroDebevere**

### CODE REVIEW

File: Protocol/Roslyn/QActionSourceText.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿namespace Skyline.DataMiner.CICD.Models.Protocol
+{
+    using System;
+    using System.Text;
+
+    using Microsoft.CodeAnalysis.Text;
+
+    using Skyline.DataMiner.CICD.Models.Protocol.Read;
+
+    internal class QActionSourceText : SourceText
```
Comment (MichielOda) [inline Protocol/Roslyn/QActionSourceText.cs:10]: Currently this is internal, but will be used in DIS for 'NavigateTo' in the validator & comparer. Unless you have thought of another way to achieve the same result in DIS?

**Review by MichielOda — CHANGES_REQUESTED** [review]
Did some general cleanup and made some stuff internal as I didn't saw it being used in DIS or SysDev.

File: Protocol/Roslyn/QActionSourceText.cs (hunk 1 above)
Comment (PedroDebevere) [reply Protocol/Roslyn/QActionSourceText.cs]: made public

**Review by MichielOda — APPROVED** [review]

**Comment by PedroDebevere** [comment]
merge

**Pull request merged**